ui/src/components/Secrets.vue

```diff
@@ -386,7 +386,7 @@ const missingSecretData = computed(() =>
     e.key !== "" &&
     (
       e.value !== "" ||
-      (e.file.length == 1 && e.file[0] instanceof Blob || e.file[0] instanceof File)
+      (e.file.length == 1 && e.file instanceof Blob || e.file instanceof File)
```

Comment:
Hm, that does not look right, either. Will have to think about it. We don't have a secret, when we have an empty array and are not instances of `Blob` or `File`, right?

Reply:
What do you think about the following code (that will ignore complete empty lines):

```javascript
const missingSecretData = computed(() =>
  secrets.value
    .filter((e) =>
      e.key !== "" || e.value !== "" || e.file instanceof Blob || e.file instanceof File
    )
    .some((e) =>
      e.key === "" ||
      (
        e.value === "" &&
        !(e.file instanceof Blob || e.file instanceof File)
      )
    )
)
```

That should probably be combined with a check, that we have at least one fully/partially filled secret.

Reply:
Rebased with master :)